stack.py

```diff
@@ -309,10 +316,22 @@ def eval_program(program):
     'gt': lambda a, b: float(b > a),
     'ge': lambda a, b: float(b >= a),
     'lt': lambda a, b: float(b < a),
-    'le': lambda a, b: float(b <= a)
+    'le': lambda a, b: float(b <= a),
+    'and': lambda a, b: 1 if (a and b) else 0,
```

Comment:
You should be able to just write `float(a and b)`, etc, since `float(True) == 1.0` and `float(False) == 0.0`.